pyracf/scripts/setup_precheck.py

```diff
@@ -0,0 +1,34 @@
+from pyracf import ResourceAdmin, SecurityRequestError
+
+
+def setup_precheck():
+    resource_admin = ResourceAdmin()
+    try:
+        access = resource_admin.get_my_access("IRR.IRRSMO00.PRECHECK", "XFACILIT")
```

Comment:
Should we use `.upper()` on `access` so that `{access}` in the console messages is uppercase for consistency?

Reply:
was just thinking this, it seems appropriate. fixed.